components/sup/src/manager/service/terminator.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2019 Chef Software Inc. and/or applicable contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use super::spawned_future::SpawnedFuture;
+use crate::sys::{service, ShutdownMethod};
+use futures::sync::oneshot;
+use habitat_core::os::process::Pid;
+use std::thread;
+
+static LOGKEY: &'static str = "ST"; // "Service Terminator"
+
+/// Shut a service process down.
+///
+/// This is performed in a separate thread in order to prevent
+/// blocking the rest of the Supervisor.
+pub fn terminate_service(pid: Pid, service_group: String) -> SpawnedFuture<ShutdownMethod> {
+    let (tx, rx) = oneshot::channel();
+
+    let handle_result = thread::Builder::new()
+        .name(format!("{}-{}", LOGKEY, pid))
+        .spawn(move || {
+            outputln!(preamble service_group, "Terminating service (PID: {:?})", pid);
```

Comment:
```suggestion
            outputln!(preamble service_group, "Terminating service (PID: {})", pid);
```
`Pid` implements `Display` on all platforms, doesn't it? This same thing occurs in several other lines.

Reply:
Looks like it... on Linux, `Pid` is ultimately an alias for `i32`, and on Windows, it's a `DWORD`, which in turn is an alias for `u32`, and `Display` is indeed implemented for both of those 😄 

We should probably get in the habit of using `{}` rather than `{:?}` for things like this that are intended to be user-facing informational output.